### DISCUSSION TITLE
How can i change the default key mapping as it's interfering with my other keymaps?

### DISCUSSION THREAD

**Discussion by imtheaman** [d99kris/nchat#253] [Q&A]
how can i change the defulat keymaps as it's interfereing with my other keymaps?

**Comment by d99kris** [comment]
Hi @imtheaman - you will need to edit `~/.nchat/key.conf` in a text editor and change the key bindings as needed. To determine the key code one can start nchat's keydump mode with `nchat -k`. Hope this helps.

**Reply by imtheaman** [reply]
Thanks @d99kris.
